packages/react-next/RELEASE_NOTES.md

```diff
@@ -165,7 +165,7 @@ TODO: Diff of OUFR vs date-time DatePicker
 
 ### Slider
 
-TODO: document any API or functionality changes
+- Deprecated prop `ariaLabel` (use `aria-label`)
```

Comment:
If this is the overall plan, we should probably make a work item to make a change like this across all the components (and ensure they handle the native prop correctly).

Reply:
good catch, i believe this change was made bc of the convergence work. I reverted the change. we can discuss in next sync to see if we should do a round of deprecations.